src/Plugins/Export/Export/ExportAssetTextDocument.php

```
@@ -0,0 +1,174 @@
+class ExportAssetTextDocument implements FromCollection, WithColumnFormatting, WithColumnWidths, WithDefaultStyles, WithHeadings, WithMapping, WithStyles
+{
+    use Exportable;
+
+    private Collection $models;
+
+    private array $locales;
+
+    private Collection $styleCollection;
+
+    // Keep track of pending drawings to be added later
+    private array $pendingDrawings;
```

PHPMD (2.15.0)

Avoid unused private fields such as '$pendingDrawings'. (Unused Code Rules)

(UnusedPrivateField)